### ISSUE TITLE
Timestamp axis on msno.matrix()

### ISSUE THREAD

**Issue by VolkrB** [ResidentMario/missingno#14]
Hey ResidentMario, cool project! My data is a well behaving time series, so I found it convenient to add a Timestamp axis option to my msno.matrix function. This helps me narrow my data to look for missing stuff more precisely. Wanted to share with you, and hear your thoughts about it.
~~~~
In [103]: rng = pd.date_range('1/1/2011', periods=144, freq='H')
In [104]: new = np.random.randn(len(rng))
In [105]: new[new>1] = np.nan
In [114]: ts = pd.DataFrame({'lol':new,'lol1':new1,'lol2':new2,'lol3':new3,'lol4':new4}, index=rng)
In [115]: msno.matrix(ts)
~~~~

![msno_example](https://cloud.githubusercontent.com/assets/20803912/18973814/b2cad2e4-865c-11e6-80d5-d3bb72e2f71e.png)

**Comment by ResidentMario** [comment]
For well-behaved time series data, this is indeed a better default, and it doesn't seem too hard to implement (shoot me a pull request!). However, a count display element should still be an option&mdash;maybe with a `timeseries=False` keyword parameter?

In general, I think the package could use a timeseries-oriented visual abstraction of some kind. But I haven't figured out what just yet.

**Comment by VolkrB** [comment]
Yea, i was thinking the same thing about the `timeseries=False`. I will need some time to work on it (I want to also implement different frequencies for the timestamps), but then I will pull request and see what your thoughts are.

**Comment by ResidentMario** [comment]
Addressed via ResidentMario/missingno#15.
